packages/manager/apps/pci-rancher/src/public/translations/pci-rancher/dashboard/Messages_fr_FR.json

```diff
@@ -16,5 +16,12 @@
   "last_update_date": "Dernière mise à jour {{date}} à {{hour}}",
   "editNameRancherSuccess": "Le nom de votre Managed Rancher service a bien été modifié.",
   "editNameRancherError": "Une erreur est survenue. Merci de réessayer ultérieurement.",
-  "comming_soon": "Bientôt disponible"
+  "comming_soon": "Bientôt disponible",
```

Comment:
Sometimes it's snake_case, sometimes it's camelCase : we should harmonize a bit within the same file at least